scripts/website/runCypress.ts

```diff
@@ -46,22 +46,22 @@ const runCypress = async ({ report, coverage }: { readonly report: boolean; read
 };
 
 // tslint:disable-next-line readonly-array
-const mutableCleanup: Array<() => Promise<void> | void> = [];
+const mutableCleanup: Array<() => void> = [];
 
 // tslint:disable-next-line no-let
 let shuttingDown = false;
 const shutdown = (exitCode: number) => {
   if (!shuttingDown) {
     shuttingDown = true;
     console.log('Shutting down...');
-    Promise.all(mutableCleanup.map((callback) => callback()))
-      .then(() => {
-        process.exit(exitCode);
-      })
-      .catch((error) => {
-        console.error(error);
-        process.exit(1);
-      });
+    try {
+      console.log(`Shutdown called with exit code ${exitCode}`);
+      mutableCleanup.forEach((callback) => callback());
+      // process.exit(exitCode);
```

Comment:
@dicarlo2 Uncommenting this line, `process.exit()`, will cause the two `execa` processes to *not* cancel. In other words, when the tests finish the website and servers would still be up if `process.exit()` was there. That seems strange. Is there a reason that `process.exit()` needs to be called here? Or can it be left out?

Reply:
I think you don't want to change this bit of code - let's chat about it today.

Reply:
Ok. Ready when you are